docs/howto/garbage-collection/committed.md

```diff
@@ -3,16 +3,18 @@ title: (deprecated) Committed Objects
 description: Clean up unnecessary objects using the garbage collection feature in lakeFS.
 parent: Garbage Collection
 grand_parent: How-To
-nav_order: 10
+nav_order: 1980
 has_children: false
+redirect:
+    - /howto/garbage-collection-committed.html
 ---
 
 # Garbage Collection: committed objects
 
 {: .warning-title }
 > Deprecation notice
 >
-> This page describes a deprecated feature. Please visit the new [garbage collection documentation](./garbage-collection.md).
+> This page describes a deprecated feature. Please visit the new [garbage collection documentation](./index.html).
```

Comment:
This PR changes many links to have an "html" extension. This means that if use an IDE that supports markdown, I will not be able to navigate using these links, and I will not be notified if the links are broken.

Reply:
@arielshaqed made a fix to use markdown throughout and this is incorporated in this PR now too.